src/encoded/static/components/blocks/richtext.js

```diff
@@ -46,9 +46,9 @@ export default class RichTextBlockView extends React.Component {
     }
 
     setupEditor() {
-        const ck = require('ckeditor4');
-        ck.disableAutoInline = true;
-        this.editor = ck.inline(this.domNode, {
+        require('ckeditor4');
+        CKEDITOR.disableAutoInline = true;
```

Comment:
To avoid the ESLint error for CKEDITOR being undefined, you have to disable the `no-undef` rule in this area. We’ve usually done it by putting `/* eslint-disable no-undef */` before the function/method, and `/* eslint-enable no-undef */` after. Sounds like ckeditor5 makes integrating with webpack better, but not sure if it changed its API too much to use here.

Reply:
Added, thanks.